### PR TITLE
[SQL] Update status for ??? npc in Dynamis - Jeuno 

### PR DESCRIPTION
**Pull request by Chronos77**
**_I affirm:_**
- [x] that I agree to Topaz Next's [Limited Contributor License Agreement](https://github.com/topaz-next/topaz/blob/release/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by zach2good** [comment]
Not a thing I particularly understand, can you peeps please take a look vs your caps? @wrenffxi @UynGH

File: sql/npc_list.sql [hunk 1]
```diff
@@ -21453,11 +21453,11 @@ INSERT INTO `npc_list` VALUES (17547501,'Achieve_Master','Achieve Master',0,0.00
...
-INSERT INTO `npc_list` VALUES (17547509,'qm0','???',0,0.000,0.000,-101.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547510,'qm1','???',0,0.000,-5.000,68.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547511,'qm2','???',0,-23.000,2.000,64.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547512,'qm3','???',0,23.000,2.000,64.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547513,'qm4','???',0,0.000,2.000,127.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17547509,'qm0','???',0,0.000,0.000,-101.000,1,50,50,0,0,0,2,3,0x000034003F3F3F00000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17547510,'qm1','???',0,0.000,-5.000,68.000,1,50,50,0,0,0,2,3,0x000034003F3F3F00000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17547511,'qm2','???',0,-23.000,2.000,64.000,1,50,50,0,0,0,2,3,0x000034003F3F3F00000000000000000000000000,0,NULL,0);
```
Comment (TeoTwawki) [inline sql/npc_list.sql:21458]: this is infilling unused bytes here. Retail doesn't zero them out so you get value sin the mask that don't mean anything. that 0x00003400 before it is the actual look and the 3f3f3f does nothing here.

Comment (Chronos77) [reply sql/npc_list.sql:21458]: if I remove the 3f3f3f, the ??? do not appear and i can't target it.

Comment (Chronos77) [reply sql/npc_list.sql:21458]: Ok, i change it and i set status to 0 otherwise it does not appear

Comment (TeoTwawki) [reply sql/npc_list.sql:21458]: status controls the state of the object, including spawned or despawned. that's the field making it not appear as a target. the model is the same in both cases here though.

Comment (TeoTwawki) [reply sql/npc_list.sql:21458]: https://github.com/topaz-next/topaz/blob/release/src/map/entities/baseentity.h#L48

**Comment by wrenffxi** [comment]
Setting status to 0 on these five QMs looks correct to me.  In Nyu's captures, they're sometimes 0/visible, and sometimes 2/hidden, but I assume their starting state should be 0/visible.  Like Teo said, the model/look of 00003400 was correct, and Chronos adjusted back to that.  So the changes in this PR to npc_list.sql are good.

However, we should not commit any of the other changes.

First, those NMs -should- have multiple mobIds.  They exist both in the POL extract and in Nyu's captures with multiple mobIds. 
 Figuring out the mechanics of which mobId pops when is part of our [Dynamis TODOs](https://github.com/topaz-next/topaz/issues/318).

Second, all evidence I can find indicates these should be spawntype 32/lottery, not 0. Example:
https://ffxiclopedia.fandom.com/wiki/Talk:Dynamis_-_Xarcabard#Post_2011_Update
"Notorious monsters are lottery spawn around there previous spawn locations."
They're all marked as Lottery on bg-wiki too:
https://www.bg-wiki.com/bg/Gabblox_Magpietongue
So I believe these NMs spawn by lottery when players kill the normal mobs in the area.  And it's possible that certain NM mobIds spawn from certain regular mobs.  My guess is that for example regular RDM mobs are the placeholders for the RDM NM.  But this is just a guess.  We need someone to study this on retail to nail down the specifics.

Third, looking at the changed files in this PR, I see it includes "Submodule FFXI_Events added at 608885".  I'm not sure what that is, but it probably doesn't belong there.

**Comment by Chronos77** [comment]
Alright @wrenffxi, thanks for informations. I fix it.

File: sql/npc_list.sql (hunk 1 above)
Comment (Chronos77) [reply sql/npc_list.sql:21458]: Ok thank you

**Review by wrenffxi — APPROVED** [review]

**Review by zach2good — APPROVED** [review]

**Pull request merged**